dist/migration/migrate-page/ab-migrate.css

```diff
@@ -0,0 +1,257 @@
+.clearfix:before, .clearfix:after {
+	content: "";
+	display: table;
+}
+
+.clearfix:after { clear: both }
+
+.clearfix { zoom: 1 }
+
+.ab-migrate * {
+	-webkit-box-sizing: border-box;
+	-moz-box-sizing: border-box;
+	box-sizing: border-box;
+}
+
+.atomic-blocks_page_atomic-blocks-migrate-page #update-nag {
+	display: none !important;
+}
+
+.atomic-blocks_page_atomic-blocks-migrate-page #wpcontent {
+	padding-left: 0;
+}
+
+.ab-migrate {
+	overflow: hidden;
+	margin: 0;
+}
+
+.ab-migrate .panel-right .theme-image {
+	background: #64717E;
+	padding: 2% 2% 0 2%;
+	display: inline-block;
+	width: 100%;
+	width: 23%;
+}
+
+.ab-migrate .panels {
+	margin: 0 0 0 0;
+}
+
+.ab-migrate .panel {
+	display: inline-block;
+	width: 100%;
+	padding: 4%;
+	background: #fff;
+	font-size: 16px;
+	animation: smoothFade .3s;
+	-moz-animation: smoothFade .3s;
+	-webkit-animation: smoothFade .3s;
+	-o-animation: smoothFade .3s;
+}
+
+@keyframes smoothFade {
+	from {
+		opacity:0;
+	}
+	to {
+		opacity:1;
+	}
+}
+
+@-moz-keyframes smoothFade {
+	from {
+		opacity:0;
+	}
+	to {
+		opacity:1;
+	}
+}
+
+@-webkit-keyframes smoothFade {
+	from {
+		opacity:0;
+	}
+	to {
+		opacity:1;
+	}
+}
+
+.ab-migrate .panel-left.visible {
+	display: inline-block;
+	animation: smoothFade .3s;
+	-moz-animation: smoothFade .3s;
+	-webkit-animation: smoothFade .3s;
+	-o-animation: smoothFade .3s;
+}
+
+.ab-migrate .panel p {
+	font-size: 19px;
+	line-height: 1.7;
+}
+
+.ab-migrate .panel-left {
+	display: inline-block;
+	display: none;
+	width: 63%;
+}
+
+.ab-migrate .panel-left img {
+	max-width: 100%;
+	height: auto;
+}
+
+.ab-migrate .intro-wrap {
+	padding: 3% 0 0 0;
+	background: #5a3fd6 url("images/geo.png");
```

Comment:
This image path is wrong and causes a 404. Looks like it lives at `/dist/getting-started/images/geo.png` instead.

Reply:
This is also fixed in #406. I'm going to merge that in here so we have the full picture.